### ISSUE TITLE
Stack not found when number of stacks is over 100

### ISSUE THREAD

**Issue by kevtaylor** [kubernetes-incubator/kube-aws#1534]
Error returned from `kube-aws apply` against an existing stack
```
generating assets for control-plane, etcd, network, nodepool-a, nodepool-b, nodepool-c
Streaming filtered Journald logs for log group 'kt'...
NOTE: Due to high initial entropy, '.service' failures may occur during the early stages of booting.
Streaming CloudFormation events for the cluster 'kt'...
Error: error updating cluster: AlreadyExistsException: Stack [kt] already exists
        status code: 400, request id: 7f709683-18ab-11e9-b0ce-a1ff802266c1
```

Switching on the verbose logging reveals that the list stacks is not paginated through all stacks,
just the first page of return values
```
testing whether cf stack kt exits
calling AWS cloudformation ListStacks ->
<- AWS Responded with 100 stacks
found no active stacks with id kt
```

**Comment by kevtaylor** [comment]
Duplicate of #1480

**Comment by kevtaylor** [comment]
This fix is not in the 12.0 or 12.1 releases
